packages/pillarbox-playlist/README.md

```diff
@@ -84,21 +84,23 @@ The following table outlines the key methods available in the this plugin:
 When initializing the playlist plugin, you can pass an `options` object that configures the
 behavior of the plugin. Here are the available options:
 
-| Option        | Type    | Default | Description                                                                                 |
-|---------------|---------|---------|---------------------------------------------------------------------------------------------|
-| `playlist`    | Array   | `[]`    | An array of playlist items to be initially loaded into the player.                          |
-| `repeat`      | Boolean | `false` | If true, the playlist will start over automatically after the last item ends.               |
-| `autoadvance` | Boolean | `false` | If enabled, the player will automatically move to the next item after the current one ends. |
+| Option                        | Type    | Default | Description                                                                                 |
+|-------------------------------|---------|---------|---------------------------------------------------------------------------------------------|
+| `playlist`                    | Array   | `[]`    | An array of playlist items to be initially loaded into the player.                          |
+| `repeat`                      | Boolean | `false` | If true, the playlist will start over automatically after the last item ends.               |
+| `autoadvance`                 | Boolean | `false` | If enabled, the player will automatically move to the next item after the current one ends. |
+| `previousNavigationThreshold` | Number  | 3       | Threshold in seconds for determining the behavior when navigating to the previous item.     |
```

Comment:
we could add this to the jsdoc of the class constructor. I don't see the default values reflected in the `options`.

Reply:
Done!